### PR TITLE
Added Logging

### PR DESCRIPTION
**Pull request by madhavajay**
## Description
expelliarmus verbosus 🧙🏽‍♂️ (expel the VERBOSE)

- sy.VERBOSE is gone
- import syft as sy # absolute import is eradicated (except for Pointer)
- new logging example code added to MNIST notebooks

## Affected Dependencies
None

## How has this been tested?
Locally

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: tests/syft/core/common/serializable_test.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # import uuid
```
Comment (Jasopaum) [inline tests/syft/core/common/serializable_test.py:1]: We could remove that

**Review by Jasopaum — APPROVED** [review]
Awesome 💣

File: tests/syft/core/common/serializable_test.py (hunk 1 above)
Comment (madhavajay) [reply tests/syft/core/common/serializable_test.py:1]: Good spot, your eyes are sharp! ❤️

**Comment by madhavajay** [comment]
This is in the syft_0.3.0 now.

**Pull request closed without merging**